src/SessionInstance.php

```diff
@@ -188,13 +198,17 @@ public function destroy()
     {
         $this->init();
 
-        # Start the session up, but ignore the error about headers already being sent
-        @session_start();
+        # Remove the session cookie
+        setcookie(
+            $this->name, "", 1, $this->cookie->getPath(), $this->cookie->getDomain(),
+            $this->cookie->isSecure(), $this->cookie->isHttpOnly()
+        );
 
-        # Clear the session data
-        $_SESSION = [];
```

Comment:
_"Only use session_unset() for older deprecated code that does not use $_SESSION"_
http://php.net/manual/en/function.session-destroy.php